app/helpers/admin/analytics_helper.rb

```diff
@@ -4,8 +4,15 @@ def track_analytics_data(type, message)
       {
         'module' => 'auto-track-event',
         'track-action' => "alert-#{type}",
-        'track-label' => message,
+        'track-label' => flash_text_without_email_addresses(message),
       }
     end
+
+    def flash_text_without_email_addresses(message)
+      text_message = strip_tags(message)
```

Comment:
We're already calling `strip_tags` further up the chain. See https://github.com/alphagov/whitehall/pull/2502/files#diff-c85d89dacb693350b479e53b8b46f36eR5